platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -128,6 +135,11 @@ public void init() throws Exception {
         return export(Paths.get(exportDirectory));
     }
 
+    @Override
+    public Collection<Describable> getInfo() {
+        return listMigratableDescribables();
```

Comment:
You mentioned in our last discussion **not** returning the entire `Migratable` but only the `Describable` because that's all the CLI/GUI actually needs. Otherwise they might fire a breaking invocation if used maliciously.